### DISCUSSION TITLE
Is it possible to skip bot commands?

### DISCUSSION THREAD

**Discussion by ABigDangMango** [victornpb/undiscord#595] [1. Questions & Answers]
I'm trying to purge my messages from a bot-heavy server. The deletion keeps going into an infinite error loop and it kinda defeats the purpose of this whole endeavor if I have to babysit it every 25 or so message to prevent it from getting rate limited over trying to delete one message over and over.

Is there a way to filter _out_ specific messages or users?

**Comment by ghost** [comment]
Yes, I got the exact same issue. It's trying to delete Dyno command bot messages.
